nilearn/datasets.py

```diff
@@ -427,7 +427,11 @@ def _fetch_file(url, data_dir, resume=True, overwrite=False,
     initial_size = 0
     try:
         # Download data
-        print 'Downloading data from %s ...' % url
+        if verbose == 0:
+            # Remove variables from url for better display
+            print 'Downloading data from %s ...' % url.split('?')[0]
```

Comment:
urllib.splitquery may be a bit clearer. It returns a tuple `url_without_query, query`

Also slight niggle, I would do 
```python

displayed_url = urllib.splitquery(url)[0] if verbose == 0 else url
print 'Dowloading data from %s ...' % displayed_url
```